throughput/config.go

```diff
@@ -36,6 +42,36 @@ func NewDefaultConfig(
 	}
 }
 
+// Default config for using the load test script.
```

Comment:
Should this just be moved to the caller since it's just creating an auth factory from the key and adding a check on the URIs? Seems the caller could run these steps and then use the existing `NewDefaultConfig`

Reply:
yea, we could move `authFactory` to be created by the caller, but the defaults are different than `NewDefaultConfig`. And the default config is for e2e testing so we want it to terminate quickly.

Reply:
updated to generate the factory in the caller

Reply:
Hmm if that's the case can we name them separately rather than calling both default? We could also separate the zipf distribution parameters from the throughput parameters as those are in fact default params.

Perhaps `NewFastConfig` and `NewLongRunningConfig` ?